creme/creme_config/bricks.py

```diff
@@ -634,25 +635,35 @@ class ButtonMenuBrick(Brick):
     template_name = 'creme_config/bricks/button-menu.html'
     configurable = False
 
-    def detailview_display(self, context):
+    @staticmethod
+    def get_buttons():
+        get_ct = ContentType.objects.get_for_id
```

Comment:
The field 'ButtonMenuItem.content_type' is a CTypeForeignKey, which automatically use ContentType.objects.get_for_id(). So this optimzation is useless.